cfme/fixtures/vm.py

```diff
@@ -59,12 +91,20 @@ def _get_vm_name(request):
 
 
 @pytest.fixture(scope="module")
-def full_template_vm_modscope(setup_provider_modscope, request, full_template_modscope, provider):
```

Comment:
These fixtures are currently in use in the framework, and can't just be removed.